.github/workflows/lint.yaml

```diff
@@ -0,0 +1,26 @@
+name: lint
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      # only on merges to master branch
+      - master
+      # and version branches, which only include minor versions (eg: v3.4)
+      - v[0-9]+.[0-9]+
```

Comment:
```suggestion
```
We don't maintain versioned branches for Ansible-st2 repository, so worth removing.